app/uk/gov/hmrc/gform/sharedmodel/TaxPeriods.scala

```diff
@@ -17,13 +17,10 @@
 package uk.gov.hmrc.gform.sharedmodel
 
 import java.util.Date
-
-import cats.data.NonEmptyList
 import julienrf.json.derived
-import play.api.data.validation.ValidationError
 import play.api.libs.json._
 import uk.gov.hmrc.gform.sharedmodel.formtemplate._
-import scala.collection.immutable.{ ::, List, Nil }
+import scala.collection.immutable.{List}
```

Comment:
Please install this plugin https://plugins.jetbrains.com/plugin/8236-scalafmt to get proper source code formatting from IntelliJ or you can start to use sbt for compilation. 

When people do not use scalafmt all other devs are potentially affected.

Btw. neither `import scala.collection.immutable.{List}` is needed

Reply:
I installed scalafmt before the most recent commit and tried to get it to format the code but it didn't seem to do anything